nebula/ui/components/VPNSubscriptionOption.qml

```diff
@@ -159,7 +152,14 @@ RadioDelegate {
                 font.pixelSize: VPNTheme.theme.fontSize
                 lineHeight: 17.68
                 lineHeightMode: Text.FixedHeight
-                text: col.subscriptionDuration !== -1 ? (col.subscriptionDuration > 1 ? col.monthlyPrice : col.productSingleMonth) : ""
+                text: {
+                    if(col.subscriptionDuration > 0) {
+                        return VPNSubscriptionData.plusTax ? VPNI18n.SubscriptionManagementPlanMonthlyPlusTax.arg(productMonthlyPrice) : VPNI18n.SubscriptionManagementPlanMonthlyWithoutTax.arg(productMonthlyPrice)
+                    }
+                    else {
```

Comment:
No `else` after `return`

Reply:
Plus why this ""? Can you simply have the first if-stmt?

Reply:
returning nothing would result in a `Unable to assign [undefined] to QString`